providers/implementations/signature/dsa_sig.c

```diff
@@ -710,11 +710,6 @@ static int dsa_common_set_ctx_params(void *vpdsactx, const OSSL_PARAM params[])
     PROV_DSA_CTX *pdsactx = (PROV_DSA_CTX *)vpdsactx;
     const OSSL_PARAM *p;
 
-    if (pdsactx == NULL)
-        return 0;
-    if (params == NULL)
-        return 1;
-
```

Comment:
Would you mind adding a comment above this function, saying something along the lines of "Any caller must check that |vpdsactx| is not NULL and |params| isn't empty before calling this function"?